boards/adafruit/feather_esp32s3/Kconfig.defconfig

```diff
@@ -0,0 +1,23 @@
+# Adafruit Feather ESP32S3 board configuration
+
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# Copyright (c) 2024 Leon Rinkel <leon@rinkel.me>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_ADAFRUIT_FEATHER_ESP32S3_ESP32S3_PROCPU
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 65535 if WIFI && BT
+	default 51200 if WIFI
+	default 40960 if BT
+	default 4096
```

Comment:
rename file as `Kconfig`

Reply:
Replace all the content in this new file to the below. Current source is old and not needed anymore:
```c
config HEAP_MEM_POOL_ADD_SIZE_BOARD
	int
	default 4096 if BOARD_ADAFRUIT_FEATHER_ESP32S3_ESP32S3_PROCPU
	default 256 if BOARD_ADAFRUIT_FEATHER_ESP32S3_ESP32S3_APPCPU
	```